mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/RowMatrix.scala

```diff
@@ -131,7 +131,7 @@ class RowMatrix(
       throw new IllegalArgumentException(s"Argument with more than 65535 cols: $cols")
     }
     if (cols > 10000) {
-      val mem = cols * cols * 8
+      val mem = (math.pow(cols, 2) * 8).formatted("%,.0f")
```

Comment:
>> 17 ? a mebibyte is 2^20 bytes so that would make sense. I don't know what the overall stance on mega vs mebibyte is in the nomenclature here but think everything is megabytes, in which case / 125000 is right.